src/EODReminder.js

```diff
@@ -34,7 +32,8 @@ const sendEODs = (reminders) => {
 }
 
 const checkHoliday = () => {
-    if (holidayDates.indexOf(new Date().toISOString().slice(0, 10)) != -1) {
+    const holidayJSON = require(path.join(__dirname, '../config-files/holidays.json'));
+    if (holidayJSON["holidays"].indexOf(new Date().toISOString().slice(0, 10)) != -1) {      
```

Comment:
An array by itself is considered valid JSON, so `holidays.json` can be: `[ date, date, ...]` instead of `{ "holidays: [...] }`. Also consider using `.includes` instead of `indexOf` to check for array members.